### ISSUE TITLE
BinarySensorDevice is deprecated in HA 0.110.0b

### ISSUE THREAD

**Issue by mgiako** [PiotrMachowski/Home-Assistant-custom-components-Burze.dzis.net#7]
Hi, new HA (beta) release and new errors...
Sorry to bother you but there is some new deprecations:

I have this log in HA 0.110.0b2:

```
Log Details (WARNING)
Logger: homeassistant.components.binary_sensor
Source: components/binary_sensor/__init__.py:174
Integration: Binary sensor (documentation, issues)
First occurred: 8:56:26 PM (4 occurrences)
Last logged: 8:56:31 PM

BinarySensorDevice is deprecated, modify BurzeDzisNetSensor to extend BinarySensorEntity
BinarySensorDevice is deprecated, modify BurzeDzisNetWarningsSensor to extend BinarySensorEntity
BinarySensorDevice is deprecated, modify BurzeDzisNetStormsNearbySensor to extend BinarySensorEntity
```
For now it's only a warning, I think that will be an error in future releases.
Hope you can solve this.
Thanks

**Comment by caiosweet** [comment]
> Hi, new HA (beta) release and new errors...
> Sorry to bother you but there is some new deprecations:
> 
> I have this log in HA 0.110.0b2:
> 
> ```
> Log Details (WARNING)
> Logger: homeassistant.components.binary_sensor
> Source: components/binary_sensor/__init__.py:174
> Integration: Binary sensor (documentation, issues)
> First occurred: 8:56:26 PM (4 occurrences)
> Last logged: 8:56:31 PM
> 
> BinarySensorDevice is deprecated, modify BurzeDzisNetSensor to extend BinarySensorEntity
> BinarySensorDevice is deprecated, modify BurzeDzisNetWarningsSensor to extend BinarySensorEntity
> BinarySensorDevice is deprecated, modify BurzeDzisNetStormsNearbySensor to extend BinarySensorEntity
> ```
> 
> For now it's only a warning, I think that will be an error in future releases.
> Hope you can solve this.
> Thanks

https://developers.home-assistant.io/blog/2020/05/14/entity-class-names

**Comment by scapman** [comment]
I do also have same error in 0.110.1:

Logger: homeassistant.components.binary_sensor
Source: components/binary_sensor/__init__.py:174
Integration: Sensore binario (documentation, issues)
First occurred: 19:37:30 (5 occurrences)
Last logged: 19:37:30

BinarySensorDevice is deprecated, modify BurzeDzisNetSensor to extend BinarySensorEntity
BinarySensorDevice is deprecated, modify BurzeDzisNetWarningsSensor to extend BinarySensorEntity
BinarySensorDevice is deprecated, modify BurzeDzisNetStormsNearbySensor to extend BinarySensorEntity

At the moment it is only a warning.

thanks
Andrea

**Comment by PiotrMachowski** [comment]
Fixed in v1.0.2
